src/mesh/Rope.js

```diff
@@ -124,6 +116,34 @@ Rope.prototype.refresh = function ()
 };
 
 /*
+ * Returns texture UVs
+ *
+ * @private
+ */
+
+Rope.prototype._getTextureUvs = function()
+{
+    if(!this._textureUvs)
+    {
+        this._textureUvs = new core.TextureUvs();
```

Comment:
This changes the hidden class of Rope since it adds a property that didn't exist after the ctor ran. Please add this property at least with a null value to the ctor.